### PR TITLE
chore: remove dependency from test package

### PR DESCRIPTION
**Pull request by kian99**
## Description

Remove a dependency the testutils/jimmtest package has on github.com/canonical/jimm/v3/cmd/jimmsrv/service

Realised this dependency shouldn't exist after reading #1455.
The function I've removed was a helper to create a `jimmsvc.Params` object. Used in 3 places:
- cmd/jimmsrv/service/service_test.go: Added a copy here, makes sense for this test to start a service.
- internal/jimmjwx/utils_test.go: Added a copy here, this test shouldn't need to start a jimm service, will refactor in a follow-up PR.
- internal/testutils/cmdtest/jimmsuite.go: Added a copy here, this makes sense as the package is creating a test jimm service for cmd tests to run against.

### CODE REVIEW

**Comment by kian99** [comment]
@ale8k This change should make #1455 easier to land.

**Review by ale8k — APPROVED** [review]
ty!

File: cmd/jimmsrv/service/service_test.go [hunk 1]
```diff
@@ -36,12 +38,33 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+// newTestJimmParams returns a set of JIMM params with sensible defaults
+// for tests. A test can override any parameter that it needs.
+// Note that newTestJimmParams will create an empty test database.
+func newTestJimmParams(t jimmtest.Tester) jimmsvc.Params {
```
Comment (SimoneDutto) [inline cmd/jimmsrv/service/service_test.go:44]: why not exporting this function and using it all around? As service_test configuration?

Comment (SimoneDutto) [reply cmd/jimmsrv/service/service_test.go:44]: ignore my comment

**Review by SimoneDutto — APPROVED** [review]

**Pull request merged**